homeassistant/components/lutron_caseta/models.py

```diff
@@ -15,5 +14,8 @@ class LutronCasetaData:
 
     bridge: Smartbridge
     bridge_device: dict[str, Any]
-    button_devices: dict[str, dict]
-    device_info_by_device_id: dict[int, DeviceInfo]
+    dr_device_id_to_keypad: dict[str, dict]
+    keypads: dict[int, Any]
+    keypad_buttons: dict[int, Any]
+    keypad_button_names_to_leap: dict[int, dict[str, int]]
+    keypad_trigger_schemas: dict[int, vol.Schema]
```

Comment:
Does it make sense to make another model for keypad data?

maybe something like

```
@dataclass
class LutronKeypadData:
    dr_device_id_to_keypad: dict[str, dict]
    keypads: dict[int, Any]
    keypad_buttons: dict[int, Any]
    keypad_button_names_to_leap: dict[int, dict[str, int]]
    keypad_trigger_schemas: dict[int, vol.Schema]
```

And then hang `keypad_data: LutronKeypadData`  off of `LutronCasetaData` ?

Reply:
Yes, I can do that

Reply:
You could probably shorten the names then as well

Reply:
done